### PR TITLE
Backend/saved stocks

### PR DESCRIPTION
**Pull request by fabiolarobles1**
Added controller and model methods to get user saved stocks by userid

### CODE REVIEW

File: backend/app.py [hunk 1]
```diff
@@ -28,6 +29,18 @@ def handle_users():
     else:
         return jsonify("METHOD NOT ALLOWED"), 405
 
+
+#####################################################################
+#                               STOCK                               #
+#####################################################################
+
+@app.route('/stocker/stock/savedStocks', methods=["GET"])
```
Comment (crobles3) [inline backend/app.py]: I believe the userid parameter must be attached at the end of the route.

File: backend/model/stock.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+   
+#####################################################################################################
+#                                       CRUD OPERATIONS                                             #
+#####################################################################################################
+
+def getUserSavedStocks(self, userid):
+    cursor = self.conn.cursor()
+    query = "select symbol from  monitors as M ,  \
+        stock as S where M.userid = %s and s.stockid = M.stockid;"
```
Comment (crobles3) [inline backend/model/stock.py]: Capitalize s from s.stockid, just for consistency and better understanding, I know it isn't actually necessary.

Comment (crobles3) [reply backend/model/stock.py]: Just an observation, the query could have been done using natural inner join but is up to you how you implement it, the current implementation works just fine.

File: backend/model/stock.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    cursor.execute(query, (userid,))
+    stocks = cursor.fetchone()
```
Comment (crobles3) [inline backend/model/stock.py]: I don't remember how the fetchone() method works exactly, but just making an assumption based on its name and what I think I remember, doesn't it returns only the first record of the result of the query or something like that?

**Review by crobles3 — CHANGES_REQUESTED** [review]

File: backend/model/stock.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+     def __init__(self):
+        self.conn = psycopg2.connect(os.environ["DATABASE_URL"], sslmode="require")
+
+   
+#####################################################################################################
+#                                       CRUD OPERATIONS                                             #
+#####################################################################################################
+
+def getUserSavedStocks(self, userid):
```
Comment (crobles3) [inline backend/model/stock.py]: Indentation

File: backend/app.py (hunk 1 above)
Comment (fabiolarobles1) [reply backend/app.py]: done

File: backend/model/stock.py (hunk 3 above)
Comment (fabiolarobles1) [reply backend/model/stock.py]: fixed

File: backend/model/stock.py (hunk 4 above)
Comment (fabiolarobles1) [reply backend/model/stock.py]: fixed

**Review by crobles3 — APPROVED** [review]

**Pull request merged**